bin/generate-podspecs.sh

```diff
@@ -12,14 +12,16 @@ command -v pod > /dev/null || ( echo Cocoapods is required to generate podspecs;
 command -v jq > /dev/null || ( echo jq is required to generate podspecs; exit 1 )
 
 WD=$(pwd)
-DEST="${WD}/react-native-gutenberg-bridge/third-party-podspecs"
+DEST="${WD}/third-party-podspecs"
```

Comment:
If we are keeping `generate-podspecs.sh` script on gb-mobile, shouldn't we remove(not move it) to the gutenberg repo?

We probably should also delete `gutenberg-monorepo/packages/react-native-bridge/third-party-podspecs`
folder from GB too.

Reply:
Both the script and the folder are deleted, the changes are in this Gutenberg PR: https://github.com/WordPress/gutenberg/pull/21332

Reply:
Thanks for the clarification.